driver/bpf/plumbing_helpers.h

```diff
@@ -284,6 +284,17 @@ static __always_inline const struct syscall_evt_pair *get_syscall_info(int id)
 	return p;
 }
 
+static __always_inline bool *is_syscall_interesting(int id)
+{
+	bool *enabled = bpf_map_lookup_elem(&interesting_syscalls_table, &id);
+
+	if (!enabled)
+
+		bpf_printk("no syscall_info for %d\n", id);
+
+	return enabled;
```

Comment:
It feels like a `bool` return of `enabled && *enabled` would feel more natural, at least to me.

Reply:
Agree on that

Reply:
+1, done, and rebased to avoid messing up commit history.